lib/domgen/jpa/templates/abstract_entity_test.java.erb

```diff
@@ -16,7 +16,9 @@ public abstract class <%= repository.jpa.abstract_entity_test_name %>
   public void preTest()
     throws Exception
   {
+    System.setProperty( "test.db.driver", "<%= repository.facet(:sql).sql_driver %>" );
```

Comment:
The code `repository.facet(:sql).sql_driver` can be just repository.sql.sql_driver